presto-docs/src/main/sphinx/sql/insert.rst

```diff
@@ -7,18 +7,19 @@ Synopsis
 
 .. code-block:: none
 
-    INSERT INTO table_name query
+    INSERT INTO table_name [ ( column [, ... ] )  query
 
 Description
 -----------
 
 Insert new rows into a table.
 
-.. note::
```

Comment:
Slightly rephrased text:

```
If the list of column names is specified, they must exactly match the list
of columns produced by the query. Each column in the table not present in the
column list will be filled with a ``null`` value. Otherwise, if the list of
columns is not specified, the columns produced by the query must exactly match
the columns in the table being inserted into.